### PR TITLE
Fixes #37443 - edit comment from host details

### PR DESCRIPTION
**Pull request by MariaAga**

### CODE REVIEW

File: webpack/assets/javascripts/react_app/components/HostDetails/InlineEdit.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+import { sprintf, translate as __ } from '../../common/I18n';
+
+export const InlineEdit = ({
+  name,
+  defaultValue,
+  hostName,
+  editPermission,
+}) => {
+  const [value, setValue] = useState(defaultValue);
+  const [isEdit, setIsEdit] = useState(false);
```
Comment (jeremylenz) [inline webpack/assets/javascripts/react_app/components/HostDetails/InlineEdit.js]:
```suggestion
  const [isEditing, setIsEditing] = useState(false);
```

File: webpack/assets/javascripts/react_app/components/HostDetails/InlineEdit.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    setIsEdit(false);
+
+    dispatch(
+      APIActions.put({
+        url: `/api/hosts/${hostName}`,
+        key: `${hostName}-${name}-EDIT`,
+        params: {
+          [name]: value,
+        },
+        successToast: () => sprintf(__('Edited %s'), name),
```
Comment (jeremylenz) [inline webpack/assets/javascripts/react_app/components/HostDetails/InlineEdit.js]:
```suggestion
        successToast: () => sprintf(__('%s saved'), name),
```

**Review by jeremylenz — COMMENTED** [review]
Tested and works as advertised :+1: 

Just a couple wording nits

**Comment by MariaAga** [comment]
Thanks @jeremylenz, I updated them

**Review by jeremylenz — APPROVED** [review]
thanks @MariaAga ! ACK

**Pull request merged**